.release-please-manifest.json

```diff
@@ -1,28 +1,28 @@
 {
   "packages/opentelemetry-api-metrics": "0.25.0",
   "packages/opentelemetry-context-async-hooks": "0.25.0",
-  "packages/opentelemetry-context-zone-peer-dep": "0.25.0",
   "packages/opentelemetry-context-zone": "0.25.0",
-  "packages/opentelemetry-core": "0.25.0",
+  "packages/opentelemetry-context-zone-peer-dep": "0.25.0",
+  "packages/opentelemetry-core": "0.26.0",
+  "packages/opentelemetry-exporter-collector": "0.26.0",
   "packages/opentelemetry-exporter-collector-grpc": "0.25.0",
-  "packages/opentelemetry-exporter-collector-proto": "0.25.0",
-  "packages/opentelemetry-exporter-collector": "0.25.0",
+  "packages/opentelemetry-exporter-collector-proto": "0.26.0",
   "packages/opentelemetry-exporter-jaeger": "0.25.0",
   "packages/opentelemetry-exporter-prometheus": "0.25.0",
   "packages/opentelemetry-exporter-zipkin": "0.25.0",
-  "packages/opentelemetry-instrumentation-fetch": "0.25.0",
-  "packages/opentelemetry-instrumentation-grpc": "0.25.0",
+  "packages/opentelemetry-instrumentation": "0.25.1",
+  "packages/opentelemetry-instrumentation-fetch": "0.25.1",
+  "packages/opentelemetry-instrumentation-grpc": "0.26.0",
```

Comment:
Is it intended that we use independent versioning of each package?
It will be quite hard to maintain the [compat matrix](https://github.com/open-telemetry/opentelemetry-js/#compatibility-matrix) as testing happens only on latest.

Till now the question "which OTEl version do you use" had a short answer.